scripts/deploy.py

```diff
@@ -62,12 +62,12 @@ def main():
         gov_default = dev.address
     else:
         gov_default = "ychad.eth"
-    gov = get_address("Yearn Governance", default=gov_default)
+    gov = get_address("Yearn Governance", default="0x16388463d60FFE0661Cf7F1f31a7D658aC790ff7")
```

Comment:
What is this address?

Reply:
strategist multisig.
these two changes are to match the latest version of `OPERATION.md`

Reply:
Please add an ENS entry for this msig and reference that instead

Reply:
I don think we have ENS on strategist msig. But it's fine. We agreed to set `gov = dev.ychad.eth` on experimental vault.

Reply:
that's a big change, to me... I was expecting strategists would gov experiments because the whole idea is that experiments drive production strategies and vaults. devs are in charge of the production protocol, strategists in charge of experiments.

Reply:
This was a long discussion and we finally agreed on ydev governs production vaults. Other stuff is up to strategists. We can keep this as it was.